code/modules/vore/weight/fitness_machines_vr.dm

```diff
@@ -129,13 +90,11 @@
 	use_power = 0
 	idle_power_usage = 0
 	active_power_usage = 0
-	var/kilograms
 
 /obj/machinery/scale/attack_hand(var/mob/living/user)
 	if(user.loc != src.loc)
 		user << "<span class='notice'>You need to be standing on top of the scale for it to work!</span>"
 		return
 	if(user.weight) //Just in case.
-		kilograms = round(text2num(user.weight),4) / 2.20463
-		user << "<span class='notice'>Your relative weight is [user.weight]lb / [kilograms]kg.</span>"
-		user.visible_message("<span class='warning'>[user]'s relative weight is [user.weight]lb / [kilograms]kg.</span>")
+		var/kilograms = round(text2num(user.weight),4) / 2.20463
+		src.visible_message("<span class='notice'>[src] displays a reading of [user.weight]lb / [kilograms]kg when [user] stands on it.</span>")
```

Comment:
It's not important, but the current standard is to not use `src` unless you need to. Like the use inside the string in `[src]` is unavoidable, but the `src.visible_message` should really just be `visible_message` (If you decide you want to change this, change the src.loc on 95 too)